tests/behat/features/operate-subsites.feature

```diff
@@ -25,10 +25,13 @@ Feature: Create and select a subsite
 
     # Add a file to the main site
     When I go to "admin/assets"
-    And I press the "Add folder" button
     And I select "Main site" from "SubsitesSelect"
+    # Ensure that the new subsite is located before adding folders
+    And I go to "admin/assets"
+    And I press the "Add folder" button
+    And I wait for 2 seconds
```

Comment:
Do you remember why this change was necessary? Feels like it could be a regression or some underlying bug we're just testing around?

Reply:
I cannot remember

Reply:
Tried reverting this locally and it passed. Please revert this as it doesn't seem to be necessary.